Analytics/Classes/SEGAnalytics.m

```diff
@@ -562,8 +562,16 @@ - (void)refreshSettings
     self.settingsRequest = [self.httpClient settingsForWriteKey:self.configuration.writeKey completionHandler:^(BOOL success, NSDictionary *settings) {
         if (success) {
             [self setCachedSettings:settings];
+        } else {
+            // TODO: If settings request fail, fall back to using just Segment integration
+            // Won't catch situation where this callback never gets called - that will get addressed separately in regular dev 
+            [self setCachedSettings:@{
+                @"integrations": @{
+                    @"Segment.io": @{ @"apiKey": self.configuration.writeKey },
```

Comment:
Eventually instead of doing setCachedSettings we should do updateIntegrationWithSettings (can't do this yet because the code references self.cachedSettings directly in a few places)>